server/coordinator/scheduler/scheduler.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
+
+package scheduler
+
+import (
+	"context"
+
+	"github.com/CeresDB/ceresmeta/server/coordinator/procedure"
+	"github.com/CeresDB/ceresmeta/server/storage"
+)
+
+type Result struct {
+	p procedure.Procedure
+	// Scheduler will give the reason than why the procedure is generated.
+	schedulerReason string
+}
```

Comment:
```suggestion
type ScheduleResult struct {
	P procedure.Procedure
	// Scheduler will give the reason than why the procedure is generated.
	Reason string
}
```

Reply:
Struct name could not start with package name, it will cause golint failed, just rename `scheduleReason` to `Reason`.

Reply:
I guess the package name is `scheduler`, which is different from `schedule`.

Reply:
And the procedure should be made public? Otherwise, the result's procedure can't be accessible to caller from the other packages.